lib/log.ts

```diff
@@ -37,7 +37,7 @@ interface Options {
   silent?: boolean
 }
 
-type ConsoleArgs = [object | string, ...string[]];
+type ConsoleArgs = [object | string, ...any[]];
```

Comment:
```suggestion
type ConsoleArgs = any[];
```

IMHO we can simply use `any[]`, since `hexo-log` is just a wrapper of `console.log`.